eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -40,28 +40,45 @@ function Get-ChangeLogEntriesFromContent {
   $changeLogEntries = [Ordered]@{}
   try {
     # walk the document, finding where the version specifiers are and creating lists
-    $changeLogEntry = $null
     foreach ($line in $changeLogContent) {
       if ($line -match $RELEASE_TITLE_REGEX) {
         $changeLogEntry = [pscustomobject]@{ 
           ReleaseVersion = $matches["version"]
           ReleaseStatus  =  $matches["releaseStatus"]
           ReleaseTitle   = "## {0} {1}" -f $matches["version"], $matches["releaseStatus"]
           ReleaseContent = @()
+          FeaturesAdded = @()
+          BreakingChanges = @()
+          KeyBugsFixed = @()
+          Fixed = @()
         }
         $changeLogEntries[$changeLogEntry.ReleaseVersion] = $changeLogEntry
       }
       else {
         if ($changeLogEntry) {
+          if ($line.Trim() -match "###\s(?<sectionName>.*)")
```

Comment:
Are we confident that `###` is the only heading level we plan to support? For our version headings we support `#+\s+`. I'm not sure yet but if there ever becomes a reason that people need different headings levels then we might need to loosen this a little. However I'm find starting with this.

Reply:
For the known headers that we are parsing out level 3. Any thing higher will start a new changelog entry. Anything lower will be parsed under the level 3 header